blocks/library/quote/index.js

```diff
@@ -74,12 +74,12 @@ registerBlock( 'core/quote', {
 		);
 	},
 
-	save( attributes ) {
+	save( { attributes } ) {
 		const { value, citation, style = 1 } = attributes;
 
 		return (
 			<blockquote className={ `blocks-quote-style-${ style }` }>
-				{ value && value.map( ( paragraph, i ) => (
+				{ value && wp.element.Children.map( value, ( paragraph, i ) => (
```

Comment:
>It's not when we insert an empty quote block, if we type inside the editable it'll not produce an array if we have a single paragraph.

That's true, but in that case we have a bit of a mismatch between the shape of state on parse, and the value that will be assigned from Editable as described in your example.

Specifically, parse will treat a single paragraph as `[ <p>foo</p> ]`, whereas in a new block, the same content would be assigned as just `<p>foo</p>`